### PR TITLE
Ruby 3.4

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

File: etc/nginx/snippets/rewrites.conf [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # NOTE rewrite uses 302 redirect by default (assuming replacement does not start with a protocol)
 # current versions
-set $current_version_sdk_c_api '3.3.2';
-set $current_version_sdk_dotnet_api '3.4.2';
+set $current_version_sdk_c_api '3.3.4';
```
Comment (osfameron) [inline etc/nginx/snippets/rewrites.conf:3]: Double checking this is deliberate? (.2 increase)

File: home/modules/contribute/examples/local-staging-antora-playbook.yml [hunk 2]
```diff
@@ -13,31 +13,29 @@ content:
...
     branches: [release/4.2, release/4.1]
   - url: ~/Couchbase/docs/docs-sdk-php/
-    branches: [release/3.2, release/3.1]
+    branches: [release/4.1, release/4.0]
   - url: ~/Couchbase/docs/docs-sdk-python/
-    branches: [release/4.0, release/3.2]
+    branches: [release/4.1, release/4.0]
   - url: ~/Couchbase/docs/docs-sdk-ruby/
-    branches: [release/3.3, release/3.2]
+    branches: [release/3.4 release/33]
```
Comment (osfameron) [inline home/modules/contribute/examples/local-staging-antora-playbook.yml:36]: should be `3.3` not `33` presumably

File: home/modules/contribute/examples/local-staging-antora-playbook.yml [hunk 3]
```diff
@@ -13,31 +13,29 @@ content:
     start_path: docs
   # NOTE server-docs is currently after other server repos so nav key wins
   - url: ~/Couchbase/docs/docs-server/
-    branches: [release/6.5, release/6.6, release/7.0]
-  - url: ~/Couchbase/docs/docs-server/
-    branches: [release/7.0]
+    branches: [release/7.1 release/7.0]
   - url: ~/Couchbase/docs/docs-sdk-common/
-    branches: [release/7.0]
+    branches: [release/7.1.2, release/7.1]
```
[the remark this answers is not in the document]
Comment (RichardSmedley) [reply home/modules/contribute/examples/local-staging-antora-playbook.yml:18]: For 3.4 releases, no Server point release  (7.1.2 in Capella - but became 7.1.3)

**Review by osfameron — APPROVED** [review]
👍

File: home/modules/contribute/examples/local-staging-antora-playbook.yml (hunk 2 above)
Comment (RichardSmedley) [reply home/modules/contribute/examples/local-staging-antora-playbook.yml]: Fixed

**Pull request merged**